app/usr/lib/systemd/system/pulp_resource_manager.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
The scripting to make links to/copies of the developer versions of these files in ```pulp-dev.py``` also need to be changed.

https://github.com/pulp/pulp/blob/3.0-dev/pulp-dev.py#L228

Since the vagrant environment doesn't install the RPM, the files have to be linked/copied manually.